### DISCUSSION TITLE
Is it possible to have a custom website search query with a shortcut word?

### DISCUSSION THREAD

**Discussion by xathian** [Flow-Launcher/Flow.Launcher#2357] [Q&A]
I very frequently use the website https://www.pcgamingwiki.com and I would love to be able to open Flow Launcher and type something like

pcgw GAME

And have it take me directly to the search results page on https://www.pcgamingwiki.com Is this possible?

**Comment by pewilliam** [comment]
I don't think they have this feature currently. Would love to be able to do it too.

**Comment by VictoriousRaptor** [comment]
In `Web Search` plugin, add a keyword `pcgw`, and set the URL to 

```
https://www.pcgamingwiki.com/w/index.php?search={q}&title=Special%3ASearch
```

**Reply by miks2021** [reply]
> In `Web Search` plugin, add a keyword `pcgw`, and set the URL to
> 
> ```
> https://www.pcgamingwiki.com/w/index.php?search={q}&title=Special%3ASearch
> ```

As of now, Web search plugin doesn't offer any customisations. Where is this functionality now?

**Comment by jjw24** [comment]
Can also use custom query hotkey to achieve this in hotkey settings section.
